### PR TITLE
osd/PrimaryLogPG: fix do_op log information

### PR DESCRIPTION
**Pull request by chenfangxian**
when print log, use __func__ instead of do_op function name

### CODE REVIEW

**Comment by gmayyyha** [comment]
You can use `git pull --rebase origin target-branch`, then the pull request does not contain any "merge" commit.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst

**Comment by gmayyyha** [comment]
[osd/PrimaryLogPG.cc](https://github.com/ceph/ceph/blob/master/src/osd/PrimaryLogPG.cc) Many methods did not use `__func__`, can put all the replacement.

> void PrimaryLogPG::do_backfill(OpRequestRef op)
{
  const MOSDPGBackfill *m = static_cast<const MOSDPGBackfill*>(op->get_req());
  assert(m->get_type() == MSG_OSD_PG_BACKFILL);
  dout(10) << "do_backfill " << *m << dendl;

**Comment by liewegas** [comment]
rebase please?

File: src/osd/PrimaryLogPG.cc [hunk 1]
```diff
@@ -1888,37 +1888,37 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
 
   // object name too long?
   if (m->get_oid().name.size() > cct->_conf->osd_max_object_name_len) {
-    dout(4) << "do_op name is longer than "
+    dout(4) << __func__ << ": name is longer than "
```
Comment (amitkumar50) [inline src/osd/PrimaryLogPG.cc]: Looks : is missed at 1st instance. Does it serve any special purpose. Can you kindly elaborate.
And not including in other instances after function name.

**Comment by liewegas** [comment]
ping

**Comment by tchaikov** [comment]
i am closing this PR. please reopen it if you get more bandwidth to continue working on it.

**Pull request closed without merging**